src/NuGet.Core/NuGet.Packaging/README.md

```diff
@@ -1 +1,60 @@
-NuGet's understanding of packages. Reading nuspec, nupkgs and package signing.
+# Nuget.Packaging
+
+NuGet.Packaging is a NuGet client SDK library that provides a set of APIs to interact with `.nupkg` and `.nuspec` files from a stram. It provides a way for developers to create and read packages and work with the package metadata.
+
+## Usage
+
+> It is strongly recommended that NuGet packages are created using the official NuGet tooling and not using this low-level API. There are a variety of characteristics important for a well-formed package and the latest version of tooling helps incorporate these best practices.
```

Comment:
```suggestion
> It is strongly recommended that NuGet packages are created using the official NuGet tooling instead of this low-level API. There are a variety of characteristics important for a well-formed package and the latest version of tooling helps incorporate these best practices.
```